package.json

```diff
@@ -58,7 +58,8 @@
   },
   "jest": {
     "preset": "./jest-preset.js",
-    "testRegex": ".*\\.test\\.js$"
+    "runner": "jest-light-runner",
```

Comment:
[note] Unless [`jest-light-runner`](https://github.com/nicolo-ribaudo/jest-light-runner), tests on Node.js 18 would fail due to segmentation fault.

Reply:
@ybiquitous Shall we use the light-runner as part of this preset to make it easier for plugin authors, i.e. add it as a dependency (rather than a dev one) and include it in [`jest-preset.js`](https://github.com/stylelint/jest-preset-stylelint/blob/stylelint-16.0.0/jest-preset.js)?

Reply:
With things being in alpha, perhaps we should handle this in the migration guide instead?

We can add something along the lines of:

---

If you get a segmentation fault while running the preset on Node.js 18, you can use [jest-light-runner](https://www.npmjs.com/package/jest-light-runner):

```diff json
export default {
  preset: "jest-preset-stylelint",
  setupFiles: ["./jest.setup.js"],
+"runner": "jest-light-runner",
};
```

---

We can see how that lands. If plugin authors find it tricky, we can revisit this.

Reply:
> We can see how that lands. If plugin authors find it tricky, we can revisit this.

That sounds good, too. 👍🏼